docs/runtime_doc.md

```diff
@@ -0,0 +1,49 @@
+# Runtime
+
+## JSON ENCODING SCHEME
+From the ink project
+
+```
+Glue:           "<>", "G<", "G>"
+
+ControlCommand: "ev", "out", "/ev", "du" "pop", "->->", "~ret", "str", "/str", "nop", 
+                "choiceCnt", "turns", "visit", "seq", "thread", "done", "end"
```

Comment:
`control_commands.rs` has a few extra commands, `readc`, `rnd`, `listInt` etc.

Reply:
I just took the doc from the Ink repo, it seems their doc is not up to date